device_backends/src/DummyInterruptTriggerAccessor.cc

```diff
@@ -3,6 +3,86 @@
 
 #include "DummyInterruptTriggerAccessor.h"
 
+#include "DummyBackendBase.h"
+
 namespace ChimeraTK {
+
+  /********************************************************************************************************************/
+
+  template<typename UserType>
+  DummyInterruptTriggerAccessor<UserType>::DummyInterruptTriggerAccessor(boost::shared_ptr<DeviceBackend> backend,
+      std::function<VersionNumber(void)> interruptTrigger, const RegisterPath& registerPathName,
+      size_t numberOfElements, size_t elementsOffset, const AccessModeFlags& flags)
+  : NDRegisterAccessor<UserType>(registerPathName, {}),
+    _backend(boost::dynamic_pointer_cast<DummyBackendBase>(backend)), _interruptTrigger(std::move(interruptTrigger)) {
+    assert(_backend); // Accessor created with wrong backend type?
+
+    if(numberOfElements > 1) {
+      throw ChimeraTK::logic_error("DUMMY_INTERRUPT accessor register can have at most one element");
+    }
+
+    if(elementsOffset != 0) {
+      throw ChimeraTK::logic_error("DUMMY_INTERRUPT accessor register cannot have any offset");
+    }
+
+    flags.checkForUnknownFlags({});
+
+    NDRegisterAccessor<UserType>::buffer_2D.resize(1);
+    NDRegisterAccessor<UserType>::buffer_2D[0].resize(1);
+    NDRegisterAccessor<UserType>::buffer_2D[0][0] = numericToUserType<UserType>(1);
+  }
+
+  /********************************************************************************************************************/
+
+  template<typename UserType>
+  bool DummyInterruptTriggerAccessor<UserType>::doWriteTransfer(ChimeraTK::VersionNumber) {
+    _interruptTrigger();
+
+    return false;
+  }
+
+  /********************************************************************************************************************/
+
+  template<typename UserType>
+  void DummyInterruptTriggerAccessor<UserType>::doPreRead(TransferType) {
```

Comment:
The implementation was like that, but conceptually this accessor is void and write-only (if I am not mistaken). It does not make sense to implement the read-functions at all.

Reply:
I think we should do that separately. (Currently, I don't have the overview what this exactly does.)

Reply:
See redmine issue #12628